### DISCUSSION TITLE
Compacted topic showing negative consumer lag after replication

### DISCUSSION THREAD

**Discussion by Sharad-SKJ052** [strimzi/strimzi-kafka-operator#9621] [Q&A]
Hi,

I'm running a multi-region Kafka setup using strimzi operator version 0.35.1 and kafka running on 3.4.0. The replication process that I have implemented is the standard MM2 between two kafka clusters.
One weird thing we have observed that for few of the compacted topics we are getting negative consumer lag after replication of consumer offsets in the target cluster which doesn't seems to be matching with the source values. Really not sure whether it is expected behaviour or not, would appreciate insights here @scholzj . 

Source cluster status
<img width="1309" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/98523649/a8353c7c-a841-4048-93c7-d0b949d6c1b5">

Target cluster status
<img width="1221" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/98523649/3bbc1fa2-aeea-4454-ba1f-677e567e420b">

**Comment by scholzj** [comment]
This is a bug / design feature of older Mirror Maker 2 version (but I do not have the Kafka JIRA at hand).

**Reply by Sharad-SKJ052** [reply]
can you please let me know if there is any mitigation for this in version 3.4 if not which version has resolved this bug

**Reply by scholzj** [reply]
I think the latest versions fixed it. But I'm not sure what version exactly I'm afraid.

**Comment by odorT** [comment]
hi @Sharad-SKJ052, out of topic, can you please tell me the name of UI tool that is shown in screenshots?
